app/web/components/LanguagePickerSelect.tsx

```diff
@@ -0,0 +1,147 @@
+import CheckIcon from "@mui/icons-material/Check";
+import {
+  Box,
+  FormControl,
+  ListItemIcon,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+  styled,
+} from "@mui/material";
+import { LANGUAGE_MAP } from "i18n/constants";
+import * as React from "react";
+
+// Styled Components
+
+/**
+ * We pass a generic type to `styled(FormControl)` so it can
+ * read our `displayMode` prop in the styling callback.
+ */
+const StyledLanguageFormControl = styled(FormControl)<{
+  displayMode?: "pill" | "rect";
```

Comment:
Could you change the naming of this to `rounded` and `squared`? Just because `pill` refers to something else.